src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -86,7 +83,7 @@
  */
 public class Database {
 
-    static final Log log = LogFactory.getLog(Database.class);
+    static final Logger log = Logger.getLogger(Database.class.getName());
```

Comment:
I think using `getName()` would be the same in these cases anyway, but I have no objection to using `getCanonicalName()` and making things easier for any future additions. Fixed in 0bbde07.